get-started/index.md

```diff
@@ -234,21 +242,17 @@ web client, continue on to [Guides]({% link guides/index.md %}) for
 guides through various common actions and workflows.
 
 ## [Uninstalling Cryostat Operator](#uninstalling-cryostat-operator)
-In order to ensure that objects created by the operator and recordings created
+In order to ensure that objects created by the Operator and recordings created
```

Comment:
Remove mentions of `Recording` Custom Resource here in the last sentence?

Also, are recordings getting cleaned up here in these steps tho?